internal/mage/engine.go

```diff
@@ -308,13 +318,42 @@ func (t Engine) Dev(ctx context.Context) error {
 	return nil
 }
 
+const cniVersion = "v1.2.0"
+
+func dnsnameBinary(c *dagger.Client, arch string) *dagger.File {
+	src := c.Git("https://github.com/vito/dnsname", dagger.GitOpts{KeepGitDir: true}).
```

Comment:
nit: could open a follow-up issue for migrating this to `dagger/dnsname` (though I suppose we pull from random repos all over the place so maybe who cares for now, whatever you prefer)

Reply:
I think I'll just pull this into the repo actually as `cmd/dnsname`, it's a pretty small amount of code and should integrate nicely into our current build stack.